_i18n/pt-br/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Um firmware de código aberto para teclados baseados em AVR e ARM"
+permalink: /pt-br/
+lang: "pt-br"
+---
+Este é um firmware de teclado baseado no firmware [tmk\_keyboard](http://github.com/tmk/tmk_keyboard) com alguns recursos úteis para os controladores Atmel AVR e ARM e, mais especificamente, na [linha de produtos OLKB](http://olkb.com), no teclado [ErgoDox EZ](http://www.ergodox-ez.com), e na [linha de produtos Clueboard](http://clueboard.co/).
+
+## Teclados equipados com QMK
+
+* [Planck](https://github.com/qmk/qmk_firmware/blob/master/keyboards/planck/)
+* [Preonic](https://github.com/qmk/qmk_firmware/blob/master/keyboards/preonic/)
+* [ErgoDox EZ](https://github.com/qmk/qmk_firmware/blob/master/keyboards/ergodox_ez/)
+* [Clueboard](https://github.com/qmk/qmk_firmware/blob/master/keyboards/clueboard/)
+* [Cluepad](https://github.com/qmk/qmk_firmware/blob/master/keyboards/clueboard/17/)
+* [Gergo](https://qmk.fm/keyboards/gergo/)
+* [Georgi](https://qmk.fm/keyboards/georgi/)
+* [Atreus](https://github.com/qmk/qmk_firmware/blob/master/keyboards/atreus/)
+
+Esses projetos usam o QMK como firmware oficial e padrão - o seu está ausente na lista? [Nos informe!](https://github.com/qmk/qmk.fm/issues/new)
+
+O QMK é compatível com [muitos outros teclados e projetos suportados pela comunidade](/keyboards/). 
+
+Você pode ler mais sobre o uso do QMK com o teclado [aqui](/powered/).
+
+## Mantedores
+
+QMK é desenvolvido e mantido por Jack Humbert da OLKB, com contribuições da comunidade e, é claro, [Hasu](https://github.com/tmk).  Os firmwares dos produtos OLKB são mantidos por [Jack Humbert](https://github.com/jackhumbert), o Ergodox EZ da [ZSA Technology Labs](https://github.com/zsa), e o Clueboard por [Zach White](https://github.com/skullydazed).
+
+## Documentação
+
+[Documentação](https://docs.qmk.fm) está hospedado em [Gitbook](https://www.gitbook.com/book/qmk/firmware/details) e [Github](https://github.com/qmk/qmk_firmware/tree/master/docs) (ambos sincronizados). Você pode solicitar alterações fazendo uma solicitação de fork e [pull request](https://github.com/qmk/qmk_firmware/pulls) ou clicando em "sugerir uma edição" em qualquer página.
```

Comment:
This is actually a mistake from the English site. Our docs are hosted on docsify, rather than Gitbook.

```suggestion
[Documentação](https://docs.qmk.fm) está hospedado em [docsify](https://docsify.js.org/) e [Github](https://github.com/qmk/qmk_firmware/tree/master/docs) (ambos sincronizados). Você pode solicitar alterações fazendo uma solicitação de fork e [pull request](https://github.com/qmk/qmk_firmware/pulls) ou clicando em "sugerir uma edição" em qualquer página.
```